cogs/tags/tags.py

```diff
@@ -381,7 +385,7 @@ async def dump(self, ctx: Context):
 
     @tag.command(name="add", aliases=["create"])
     @commands.guild_only()
-    @role_or_mod_or_permissions(role=ALLOWED_ROLE, manage_messages=True)
+    @role_or_mod_or_permissions(isAllowedRole(role), manage_messages=True)
```

Comment:
`role` is a kwarg to the decorator in `rolechecks.py`, it's not something you can pass in.

Reply:
Resolved in latest commit